### PR TITLE
Fix Message when reloading multiple scripts with an error.

### PR DESCRIPTION
**Pull request by Asleeepp**
### Description

Noticed while updating my server (and i had some errors) that when i reloaded an entire folder, the "in" was colored green, which looked pretty strange. this pr fixes that. 
Although, not sure if lime is even used much across Skript (apart from reloading correctly), so maybe the lime color should just be changed overall. lmk!

Not sure if you guys want it in the other langs too, but im not sure how i could do that, im not a polyglot 🤷 

---
image

**Target Minecraft Versions:** any 
**Requirements:** none 
**Related Issues:** none

### CODE REVIEW

**Comment by sovdeeth** [comment]
you should just change the light green to light red. nothing should be green in that message imo

**Comment by Asleeepp** [comment]
> you should just change the light green to light red. nothing should be green in that message imo

would you like me to do it for the other langs too? since its just a color change should be fairly simple

**Review by Moderocky — CHANGES_REQUESTED** [review]
This was made green because of #6136.
Unfortunately, we didn't know at the time it's also used in another place (which was presumably why it was left white to begin with).

You're either going to have to separate it into two language nodes or find some colour that works for both.

**Review by Moderocky — APPROVED** [review]

**Review by Efnilite — APPROVED** [review]

**Review by UnderscoreTud — APPROVED** [review]

File: src/main/resources/lang/french.lang [hunk 1]
```diff
@@ -68,7 +68,8 @@ skript command:
 		aliases: les alias
 		script: <gold>%s<reset>
 		scripts in folder: tous les scripts dans <gold>%s<reset>
-		x scripts in folder: <gold>%2$s <lime>script¦¦s¦ dans <gold>%1$s<reset>
+		x scripts in folder success: <gold>%2$s <lime>script¦¦s¦ dans <gold>%1$s<reset>
+		x scripts in folder success: <gold>%2$s <light red>script¦¦s¦ dans <gold>%1$s<reset>
```
Comment (APickledWalrus) [inline src/main/resources/lang/french.lang:72]:
```suggestion
		x scripts in folder errors: <gold>%2$s <light red>script¦¦s¦ dans <gold>%1$s<reset>
```
oops!

**Review by APickledWalrus — APPROVED** [review]
One minor thing

**Pull request merged**